flake-modules/dev/list-plugins/list-plugins.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python3
-
```

Comment:
I believe this can stay. Sure, `writePython3Bin` will add its own, but shell interpreters will only use the first and others will be treated as "normal" comments